src/behavior-considered-undefined.md

```diff
@@ -39,6 +39,12 @@ code.
   * A value in a `char` which is a surrogate or above `char::MAX`.
   * Non-UTF-8 byte sequences in a `str`.
 
+> **Note**: Undefined behavior affects the entire program. For example, calling
+> a function in `C` that exhibits undefined behavior of `C` means your entire
```

Comment:
I don't think we usually put C/Rust in `code font`, do we?

Reply:
Ping from triage: @Havvy any thoughts on Ralf's question?

Reply:
This is correct. I've de-coded them.